src/Annotations/ReturnAnnotation.cpp

```diff
@@ -2,42 +2,32 @@
 
 ReturnAnnotation::ReturnAnnotation(AnnotationType annotationType,
                                    std::set<std::string> annotationMethods,
-                                   std::string targetName, std::string field) {
+                                   std::string targetName, int field) {
   this->annotationType = annotationType;
   this->annotationMethods = annotationMethods;
   this->targetName = targetName;
   this->field = field;
-  this->hasField = (field != "");
+  this->hasField = (field != -1);
   this->isVerified = false;
 }
 
 std::string ReturnAnnotation::generateStringRep() {
-  std::string annoTypeString;
-  std::string annoMethodsString = "{";
-
-  if (annotationType == AnnotationType::CallsAnnotation) {
-    annoTypeString = "Calls";
-  } else if (annotationType == AnnotationType::MustCallAnnotation) {
-    annoTypeString = "MustCall";
-  }
-
-  for (std::string method : this->annotationMethods) {
-    annoMethodsString += method + ", ";
-  }
-  annoMethodsString += "}";
+  std::string annoTypeString = annotationTypeToString(this->annotationType);
+  std::string annoMethodsString =
+      dataflow::setToString(this->annotationMethods);
 
   std::string fieldString;
-  if (this->field.size() > 0) {
-    fieldString = "Field = " + this->field;
+  if (this->field != -1) {
+    fieldString = "Field = " + std::to_string(this->field);
   }
 
   return "@" + annoTypeString +
          " ReturnAnnotation Function Name = " + this->targetName + " " +
          fieldString + " methods = " + annoMethodsString;
 }
 
-bool ReturnAnnotation::fieldNameEquals(const std::string &field) {
-  return field.compare(this->field) == 0;
+bool ReturnAnnotation::fieldNameEquals(int field) {
```

Comment:
This no longer checks name equality, so the name of the method should change, too.